### PR TITLE
Fixes #44 update to latest runner output

### LINKED ISSUE #44: Update to latest dump from runner
including excel export and nomenclature

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
Exporting with renamed output. Also project is also exported again as json. Good way to check that everything was performed as expected
Also colors saved as HEX files

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 1]
```diff
@@ -137,7 +137,7 @@
               "X": "Time",
               "Y": "S2|Organism|ArterialBlood|BloodCells|Midazolam|Concentration in container",
               "CurveOptions": {
-                "Color": "Green",
+                "Color": "#008000",
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: Did the colour change in example?
Just wondering why `Green `was translated to `008000 `and not to `00FF00`

Comment (msevestre) [reply examples/minimal/reporting engine input/report-configuration-plan.json]: Very good question and no idea. Is 008000 also green?

Comment (msevestre) [reply examples/minimal/reporting engine input/report-configuration-plan.json]: Lol wrong person to ask I just realized. Looking at a tool to convert I think this is the right value. This is a dark green whereas 00FF00 is a lighter green
Another reason not to use named color

Comment (msevestre) [reply examples/minimal/reporting engine input/report-configuration-plan.json]: image

Also green in .NET  is not 0 255 0. It's 0 128 0 
All good

Comment (Yuri05) [reply examples/minimal/reporting engine input/report-configuration-plan.json]: >Also green in .NET is not 0 255 0. It's 0 128 0

Seems .NET was programmed by people like me, lol

**Pull request merged**